installer/bundle/bundle_skel.sh

```diff
@@ -114,6 +121,23 @@ source_references()
 EOF
 }
 
```

Comment:
please ensure cleanup is done during uninstall

Reply:
Purge script appears to already remove `/opt/microsoft/omsagent` folder (which includes all of the code for the troubleshooter)

Location in uninstall.sh script:
https://github.com/microsoft/OMS-Agent-for-Linux/blob/9e4c1efb9eaeeafd44d5b9e05c75828534976bd0/installer/scripts/uninstall.sh#L150

Location in bundle_skel.sh script (called if uninstall.sh script isn't found):
https://github.com/microsoft/OMS-Agent-for-Linux/blob/9e4c1efb9eaeeafd44d5b9e05c75828534976bd0/installer/bundle/bundle_skel.sh#L1143